client/src/components/JobInformation/JobInformation.vue

```diff
@@ -47,7 +65,11 @@
                 </tr>
                 <tr v-if="job && job.id">
                     <td>Job API ID:</td>
-                    <td id="encoded-job-id">{{ job.id }} <decoded-id :id="job.id" /></td>
+                    <td id="encoded-job-id">
+                        {{ job.id }} <decoded-id :id="job.id" /> (<a :href="'/root?job_id=' + job.id" target="_top"
```

Comment:
Can you make the rerun a button (like it is in the history panel)

Reply:
addressed